apps/admin/lib/api/api.ts

```diff
@@ -32,9 +32,9 @@ axiosInstance.interceptors.response.use(
   async error => {
     const { response } = error;
     if (response?.data?.message) {
-      console.error('error: ', response.status, response.data.message);
+      console.error('error: ', response.data.message);
```

Comment:
I'm not against refactoring this but can you figure out a better way to do it? Plain axios errors are extremely verbose and rarely useful, that's why previously we had at status and path to give us a better sense of what went wrong. Maybe a different combination of the error fields will produce better results.  Also in general printing the status helps